.github/scripts/trymerge.py

```diff
@@ -996,8 +998,23 @@ def find_matching_merge_rule(pr: GitHubPR,
     """Returns merge rule matching to this pr or raises an exception"""
     changed_files = pr.get_changed_files()
     approved_by = set(pr.get_approved_by())
-    rules = read_merge_rules(repo, pr.org, pr.project)
-    reject_reason = f"PR {pr.pr_num} does not match merge rules"
+
+    issue_link = (f"https://github.com/{pr.org}/{pr.project}/issues/new"
+                  f"?labels=module%3A%20ci&template=bug-report.yml")
+    bug_report = (f"Please help file a [bug report]({issue_link}) "
+                  f"to PyTorch DevX Team with the PR number.")
+
+    try:
+        # Any error here means that the script couldn't load the merge rules for
+        # some reasons, and the error should say so
+        rules = read_merge_rules(repo, pr.org, pr.project)
+    except FileNotFoundError as error:
+        raise RuntimeError(str(error))
+    except RuntimeError as error:
+        raise RuntimeError(f"{error}. {bug_report}")
+
+    reject_reason = (f"PR {pr.pr_num} does not match any merge rules. "
+                     f"This should not happen. {bug_report}")
```

Comment:
Why? If PR is not review this happens left and right...

Reply:
> Why? If PR is not review this happens left and right...

Hmm I think there is already another message if the PR is not reviewed [here](https://github.com/pytorch/pytorch/blob/master/.github/scripts/trymerge.py#L1025).  This default reject message should never happen if merge_rule and trymerge is setup correctly.

Let me rewrite the message though if you think asking people to submit a bug report here is overkill.